<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20767" started-at="2020-08-08T11:15:00 UYT" finished-at="2020-08-08T11:15:21 UYT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20767" started-at="2020-08-08T11:15:00 UYT" finished-at="2020-08-08T11:15:21 UYT">
      <class name="Tests.ValidationRuleTest">
        <test-method status="PASS" signature="InitializeDriverAndLoginPage()[pri:0, instance:Tests.ValidationRuleTest@77f1baf5]" name="InitializeDriverAndLoginPage" is-config="true" duration-ms="20710" started-at="2020-08-08T11:15:01 UYT" finished-at="2020-08-08T11:15:21 UYT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- InitializeDriverAndLoginPage -->
        <test-method status="PASS" signature="initValidationRuleProperties()[pri:0, instance:Tests.ValidationRuleTest@77f1baf5]" name="initValidationRuleProperties" is-config="true" duration-ms="1" started-at="2020-08-08T11:15:21 UYT" finished-at="2020-08-08T11:15:21 UYT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initValidationRuleProperties -->
        <test-method status="FAIL" signature="correctFilledFields()[pri:0, instance:Tests.ValidationRuleTest@77f1baf5]" name="correctFilledFields" duration-ms="5" started-at="2020-08-08T11:15:21 UYT" finished-at="2020-08-08T11:15:21 UYT">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Tests.ValidationRuleTest.correctFilledFields(ValidationRuleTest.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- correctFilledFields -->
      </class> <!-- Tests.ValidationRuleTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
